build.sc

```diff
@@ -533,7 +533,7 @@ object scalalib extends MillModule {
 
 object scalajslib extends MillModule {
 
-  override def moduleDeps = Seq(scalalib, scalajslib.api)
+  override def moduleDeps = Seq(scalalib, scalajslib.`worker-api`)
```

Comment:
Moved the code of `api` inside `scalajslib` and created a new internal module called `worker-api`.